pkg/api/controller.go

```diff
@@ -2996,6 +3014,36 @@ func paginationAmount(v *PaginationAmount) int {
 	return i
 }
 
+func resolvePathList(objectList *[]string, prefixList *[]string) []catalog.PathRecord {
```

Comment:
it's up to you but this is a suggestion 
```suggestion
func resolvePathList(objects *[]string, prefixes *[]string) []catalog.PathRecord {
```